src/pybind/mgr/prometheus/module.py

```diff
@@ -539,6 +548,12 @@ def get_metadata_and_osd_status(self):
                 (service_id, hostname, version)
             )
 
+    def get_num_objects(self):
+        pg_sum = self.get('pg_dump')['pg_stats_sum']['stat_sum']
```

Comment:
Can you get what you need from `pg_summary` instead of `pg_dump`?  If not, let's add it there rather than doing the full `pg_dump` -- it's potentially a very large structure.

Reply:
Unfortunately no, it is not there, that is why I polled pg_dump. I agree it is a pretty large structure. It would also help if we could poll for a substructure from ceph-mgr, the stat_sum itself is pretty small. Unfortunately, I am not too familiar with the C++ code so I could use some help with exposing it there.